website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md

```diff
@@ -299,9 +311,9 @@ Developer accounts will be able to query the Proxy Server using SQL, but will no
  <code>Object 'DATABASE.SCHEMA.TESTING_EPHEMERAL does not exist or not authorized.</code><br></br>
```

Comment:
Can we change the name here? Only realizing right now that it is an error message from me testing something. We could call it DATABASE.SCHEMA.METRIC_MODEL_TABLE?